pkg/ecctl/init.go

```diff
@@ -102,7 +103,9 @@ const (
 	//nolint
 	passMsg = "Type in your password: "
 
-	validCredentialsMsg = "Your credentials seem to be valid, and show you're authenticated as \"%s\".\n\n"
+	validCredentialsMsg            = "Your credentials seem to be valid, and show you're authenticated as \"%s\".\n\n"
+	validCredentialsAlternativeMsg = "Your credentials seem to be valid.\n\n"
+	invalidCredentialsMsg          = "your credentials couldn't be validated make sure they're correct and try again"
```

Comment:
```suggestion
	invalidCredentialsMsg          = "your credentials couldn't be validated. Make sure they're correct and try again"
```

Reply:
Actually, I'll make the change and skip the linting for this error, otherwise the linter will 💥